code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1449,3 +1443,91 @@ var/global/list/obj/item/device/pda/PDAs = list()
 /obj/item/device/pda/emp_act(severity)
 	for(var/atom/A in src)
 		A.emp_act(severity)
+
+
+
+//Light functions, to break stacking and support diona checking
+//===============================================================
+var/mob/living/carbon/holder//The person who's holding this PDA

```

Comment:
The way you declared these, these are now global variables. I don't imagine this is what you meant?